smug.go

```diff
@@ -57,6 +59,70 @@ func (smug Smug) switchOrAttach(sessionName string, attach bool, insideTmuxSessi
 	return nil
 }
 
+//Exists verifies if file exists
+func Exists(path string) (bool, error) {
+	_, err := os.Stat(path)
+	if err == nil {
+		return true, nil
+	}
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+	return false, err
+}
+
+//ConfigPath returns a string of the full filepath
+func ConfigPath(options Options) string {
+	userConfigDir := filepath.Join(ExpandPath("~/"), ".config/smug")
+
+	var configPath string
+	if options.Config != "" {
+		configPath = options.Config
+	} else {
+		configPath = filepath.Join(userConfigDir, options.Project+".yml")
+	}
+	return configPath
+}
+
+//Create creates a new file with name options.project, unless it already exists
+func (smug Smug) Create(options Options) error {
+	path := ConfigPath(options)
+
+	t, err := Exists(path)
+	if t {
+		fmt.Println("File Already Exists", path)
+	} else {
+		f, err := os.Create(path)
+		fmt.Println("File Created!", path)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
1. Remove print and log from here
2. Just do `return err` after `os.Create`
3. If you wanna close the file later, just do `defer f.Close()`